common/install_openblas.sh

```diff
@@ -7,6 +7,7 @@ git clone https://github.com/OpenMathLib/OpenBLAS.git -b v0.3.25 --depth 1 --sha
 
 
 OPENBLAS_BUILD_FLAGS="
+NOFORTRAN=1
```

Comment:
This file is called also from [manywheel/Dockerfile_aarch64](https://github.com/pytorch/builder/blob/51d4bfff97cb0089f7c9f5adaab01017681bda14/manywheel/Dockerfile_aarch64#L83) should we instead set this flag in Dockerfile aarch64 cuda ?

Reply:
@tinglvv  Please make this script do if else, according to whether Dockerfile called this file with an argument or not. If yes, then it is arm cuda dockerfile calling. Otherwise, it is arm cpu dockerfile calling. Thanks!